motionplan/tpspace/ptgCC.go

```diff
@@ -0,0 +1,58 @@
+package tpspace
+
+import (
+	"math"
+
+	rutils "go.viam.com/rdk/utils"
+)
+
+// This does something with circles
+// Other ptgs will be based on this ptg somehow.
+type ptgDiffDriveCC struct {
```

Comment:
I don't know what ptgDiffDriveC, ptgDiffDriveCC, etc are

Reply:
They are just names of the different trajectories, with the names themselves taken from the example impl. ptgDiffDriveCC = a PTG for a diff drive or ackerman steered vehicle, whose trajectories are a Circle plus another Circle.

It might be helpful if I also add visualizations of all PTG trajectories to the repo.

Reply:
Thank you! Even adding that one-line description would be helpful.

Reply:
Added descriptions to each of the PTG structs